### DISCUSSION TITLE
Welcome to local-abbfreeathome-hass Discussions!

### DISCUSSION THREAD

**Discussion by kingsleyadam** [kingsleyadam/local-abbfreeathome-hass#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nrenner73** [comment]
Hey there,

I´m using free@home for about three years now and went with the integration by jheling so far. Recently I also experimented with the matter addon but I fear the slow progress there. So I appreciate your effort to create an integration via the local API very much.

I´m not the best programmer and not a python guy at all but at least I understand a good part of the code.

In my installation I have mostly wireless devices, including dimmimg actors, single and double relay switches, sensors without actor, room thermostats and blind/shutter actors. At the moment I´m experimenting with the new wireless flex switches and the wired switching actor SA-M-4.16.2.12 by Busch Jaeger.

So if I can be of any help in providing device configurations I´m happy to contribute!

**Reply by derjoerg** [reply]
Hey @nrenner73 
If you still have devices, which are not discovered at all or wrongly discovered, don't hesitate to open an issue for each.

**Reply by nrenner73** [reply]
Hi @derjoerg,

at the moment, all my devices are recognized by the integration and everything is working smoothly since months.

I´m sure, as soon as ABB/BJ introduce some new devices, I can´t help myself but buy/test and open issues here ;-)

BR and thanks for all your work here!
